docs/python-examples/fixtures/test_fixtures.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def alter_pvc(pvc):
+    """
+    Dummy helper function to alter pvc
+
+    Args:
+        pvc (PVC): PVC object reference
+    """
+    logger.info(f"Doing something with pvc: {pvc}")
+
+
+@pytest.fixture(scope='class')
+def three_pvcs(create_pvcs, storage_class):
+    """
+    This is fixture related just to one specific test in this module so no
+    need to have it defined in conftest
+    """
+    return create_pvcs(3, storage_class)
+
+
+class TestCreatingPVCsFromTest:
+    # If the test suite needs to share some resources it's possible to do it
+    # via class member like below, when you fill resources from test case.
+    # We just need to document that test suit needs run like whole cause if
+    # some test case depends on resources created by other test case in same 
+    # test suite (class) it's not possible to run just last test case.
+    shared_pvcs = []
+
+    @pytest.mark.parametrize("pvcs_number", (2, 4))
+    def test_create_pvcs(self, pvcs_number, create_pvcs, storage_class):
+        """
+        You can access all needed resources via fixtures as parameters in
+        method definition like in example above (create_pvcs or
+        storage_class).
+        """
+        pvcs_created = create_pvcs(pvcs_number, storage_class)
+        len(pvcs_created) == pvcs_number
```

Comment:
did you mean to ``assert`` here?

Reply:
Yep, will fix in next commit once I will send.